### ISSUE TITLE
Mac OS X Support?

### ISSUE THREAD

**Issue by SanktyRG** [GerbilSoft/rom-properties#209]
I'm really sorry if this has been asked or discussed before, I searched for any info about it and couldn't find anything, I was wondering if it would be possible to add Mac support, if it is not possible then no problem but if it is, I can volunteer to help debug and test.

I love this thumbnailer, and would love to see it available for as many people as possible.

**Comment by Masamune3210** [comment]
Does macos even support external thumbnailers and property sheets?

**Comment by GerbilSoft** [comment]
There is definitely support for external thumbnailers (Quick Look plugins). No idea about property sheets, though.

The biggest issue here is access to recent Mac hardware. The last Mac I used is a Power Mac G4 MDD running OS X 10.5. I have a Core2 Duo MacBook Pro somewhere but don't have an AC adapter for it.

**Comment by SanktyRG** [comment]
Yes, there is support for external thumbnailers and I think there is property sheets but I'm just assuming from what I've seen.

I understand access to Mac hardware is hard, I have this Macbook Pro mid 2012 with Mac OS 10.14, don't feel like updatin to 10.15 yet though. I really wish there would be a way to get your thumbnailer, it makes backup folders look really cool.

**Comment by mkeseattle** [comment]
@GerbilSoft 

Thank you for maintaining this fantastic project! 🙌 We’re curious if macOS support might ever be considered, given the community interest.

**Comment by GerbilSoft** [comment]
I don't have any reasonably current Macs, so this would mostly depend on someone contributing by writing a UI frontend. The UI frontends are rather big, though...

I'll note that rpcli *should* mostly work on macOS, at least.

**Comment by mkeseattle** [comment]
Thanks for the info. I appreciate the heads-up on rpcli working on macOS. If anyone’s up for tackling the UI frontend, that’d be awesome, but I understand it’s a hefty task.
